awssqs/config/201-clusterrole.yaml

```diff
@@ -77,7 +77,7 @@ rules:
 - apiGroups:
 # TODO on these and the eventing ones that this should be replaced with
 # an aggregated role for watching Addressables.
-  - eventing.knative.dev
+  - messaging.knative.dev
```

Comment:
instead of stashing that in here,

let's look at a ClusterRoleBinding for `addressable`, like here:

https://github.com/knative/eventing-contrib/blob/master/kafka/source/config/202-clusterrolebinding.yaml#L32-L47

that way we do leverage those that are defined there, aggregated

Reply:
@matzew aha thank you! Fixed.